### PR TITLE
Fix issue #74176

### PR DESCRIPTION
**Pull request by c2po**
As possible successor of fileupload feature similar to FEuserHomePath implemented.
In configuration record select a field from FE user which is created as subdirectory.

pluploadfe_configuration

### CODE REVIEW

**Comment by fnagel** [comment]
Otherwise looks good! Again, thanks a lot for your contribution!

File: Configuration/TCA/tx_pluploadfe_config.php [hunk 1]
```diff
@@ -87,6 +87,25 @@
 				'default' => 1,
 			)
 		),
+		'feuser_field' => array(
+			'exclude' => 1,
+			'displayCond' => 'FIELD:feuser_required:REQ:true',
+			'label' => 'LLL:EXT:pluploadfe/locallang_db.xml:tx_pluploadfe_config.feuser_field',
+			'config' => array(
+				'type' => 'select',
```
Comment (fnagel) [inline Configuration/TCA/tx_pluploadfe_config.php:95]: Missing renderType for newer TYPO3 versions:

```
'renderType' => 'selectSingle',
```

**Comment by fnagel** [comment]
Merged in f27ece4fed45a5b6d0eb050b7c40b403b77133cf

**Comment by fnagel** [comment]
Any feedback on the merge?

File: Configuration/TCA/tx_pluploadfe_config.php (hunk 1 above)
Comment (c2po) [inline Configuration/TCA/tx_pluploadfe_config.php:95]: Thank you for fixing :+1:

**Comment by c2po** [comment]
Thank you very much for the fast merge.
I am sorry for my late response and added some comments.

**Comment by fnagel** [comment]
Nice, thanks again. 

Did you test the latest master? Should work as expected but some feedback before the next TER release would be nice.

**Pull request closed without merging**